app/assets/javascripts/hubstats/deploys.js

```diff
@@ -0,0 +1,78 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
+
+$(document).ready(function() {
+  queryParameters = getUrlVars();
+  setDefaults(queryParameters);
+  activeLabels(queryParameters);
+  //initLabels(queryParameters)
+  changeColors();
+
+  $("#state-group > .btn").on("click", function(){
+    updateQueryStringParameter(queryParameters,"state",$(this).attr('id'));
+  });
+
+  $("#sort-group > .btn").on("click", function(){
+    updateQueryStringParameter(queryParameters,"order",$(this).attr('id'));
+  });
+
+  $("#group-by").on("change", function(){
+    updateQueryStringParameter(queryParameters,"group",$(this)[0].value);
+  });
+
+  $("#repos").change(function() {
+    var ids = $("#repos").val();
+    updateQueryStringParameter(queryParameters,"repos",ids);
+  });
+  
+  $("#users").change(function() {
+    var ids = $("#users").val();
+    updateQueryStringParameter(queryParameters,"users",ids);
+  });
+});
+
+function updateQueryStringParameter(queryParameters, key, value) {
```

Comment:
Are these function definitions duplicate of the ones in `pull_requests.js`?  If so, there is no need to duplicate them.  If they are different, you need to name them differently so they don't conflict.  The last one loaded will win the battle.